multicluster/apis/multicluster/v1alpha1/multiclusterconfig_types.go

```diff
@@ -44,6 +44,13 @@ type MultiClusterConfig struct {
 	// The precedence about which IP address (internal or external IP) of Node is preferred to
 	// be used as the cross-cluster tunnel endpoint. if not specified, internal IP will be chosen.
 	GatewayIPPrecedence Precedence `json:"gatewayIPPrecedence,omitempty"`
+	// The type of IP address (ClusterIP or PodIP) to be used as the Multi-cluster
+	// Services' Endpoints. All member clusters should use the same type in a ClusterSet.
+	// Defaults to ClusterIP. Existing ServiceExports should be re-exported after changing
```

Comment:
Move "Defaults to ClusterIP." to be either the 2nd sentence (after "The type of IP address (ClusterIP or PodIP) to be used as the Multi-cluster Services' Endpoints.") or the last one.

Reply:
Done, move to 2nd sentence.